docker-compose.yml

```diff
@@ -47,5 +52,13 @@ services:
       - /sys/kernel/tracing:/sys/kernel/tracing:rw
       - /sys/kernel/debug:/sys/kernel/debug:rw
       - /sys/kernel/security:/sys/kernel/security:ro
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+      # Docker socket for dockerprobe
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      # Shared volume for Docker enrichment data
+      - docker-enrichment:/enrichment:rw
```

Comment:
Please add a comment explaining why we need this to a newcomer (=Beyla doesn't return Docker-specific container labels, just IPs)

💄 "docker-metadata" ? 🙏